Fix Duplicator Pro conflict

Crabcyborg opened the pull request:
Fixes https://github.com/Strategy11/formidable-pro/issues/3459

They use Popper 2 but Bootstrap 4 uses Popper 1. De-registering it so it registers our popper js instead fixes the issue.

stephywells reviewed (changes requested):
This looks like a great suggestion. I'm thinking this might break their stuff though. Maybe we can only deregister on Formidable pages?

Crabcyborg commented:
> This looks like a great suggestion. I'm thinking this might break their stuff though. Maybe we can only deregister on Formidable pages?

Thanks @stephywells good point. I've updated it so it only calls this when it is enqueuing `formidable_admin.js` now. I moved the register popper code a bit so it could quickly register popper 1 after de-registering popper 2.

stephywells reviewed (commented):
Thank you!
